pkg/controller/cloud/node_controller.go

```diff
@@ -36,11 +36,26 @@ import (
 	clientretry "k8s.io/client-go/util/retry"
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog"
-	kubeletapis "k8s.io/kubernetes/pkg/kubelet/apis"
-	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
```

Comment:
see discussion on moving the external-facing pieces from this package to a consumable location in https://github.com/kubernetes/kubernetes/issues/82390

Reply:
@liggitt do you have a concrete idea of where to hold this kind of const strings? For example, under apimachinery path?